go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	emperror.dev/errors v0.8.1
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
-	github.com/cisco-open/operator-tools v0.36.0
+	github.com/cisco-open/operator-tools v0.36.1-0.20241011092844-539087c4e686
```

Comment:
We may want to release a newer operator-tools version, but I would not block this PR just for that.
What do you think @pepov?

Reply:
thanks for pointing out, just did: https://github.com/cisco-open/operator-tools/releases/tag/v0.37.0